centreon/www/front_src/src/Dashboards/Dashboard/api/decoders.ts

```diff
@@ -49,7 +49,7 @@ export const dashboardDetailsDecoder = JsonDecoder.object<DashboardDetails>(
       'CreatedBy By'
     ),
     description: JsonDecoder.nullable(JsonDecoder.string),
-    panels: panelsDetailsDecoder,
+    panels: JsonDecoder.optional(panelsDetailsDecoder),

```

Comment:
No, the panel property is not optional. Arnaud hasn't implemented it yet
```suggestion
    panels: panelsDetailsDecoder,
```

Reply:
ok fine, having it optional till that PR is merged allows to retrieve the current available data, as it was simply throwing errors and not retrieving the name + description